test/component/uk/gov/hmrc/individualsmatchingapi/controllers/v2/PrivilegedCitizenMatchingControllerSpec.scala

```diff
@@ -54,17 +66,31 @@ class PrivilegedCitizenMatchingControllerSpec extends BaseSpec {
         .headers(requestHeaders(acceptHeaderP2))
         .asString
 
-      Then("The response status should be 500")
-      response.code shouldBe Status.INTERNAL_SERVER_ERROR
+      Then("The response status should be 200 (Ok)")
+      response.code shouldBe OK
 
       And("The response contains a valid payload")
-      response.body shouldBe "{\"statusCode\":500,\"message\":\"NOT_IMPLEMENTED\"}"
+      parse(response.body) shouldBe parse(
+        s"""
+           {
+             "_links": {
+               "individual": {
+                 "href": "/individuals/matching/$sandboxMatchId",
+                 "name": "GET",
+                 "title": "Individual Details"
```

Comment:
Title may change to something less conflicting with Individuals Details after a chat with Lisa.